src/uwtools/drivers/driver.py

```diff
@@ -393,6 +393,15 @@ def runscript(self):
         yield None
         self._write_runscript(path)
 
+    @external
+    def show_output(self):
+        """
+        Show the output to be created by this component.
+        """
+        yield self.taskname("expected output")
+        print(json.dumps(self.output, indent=2, sort_keys=True))
+        yield asset(None, lambda: True)
```

Comment:
This task method will allow a CLI invocation like
```
uw ungrib show_output --cycle 2024-10-22T12 --config-file config.yaml
```
to display the component-output information as JSON on `stdout`, for inspection or programmatic consumption by CLI users. See the unit tests for an example.